### PR TITLE
feat(COR-1420): Added required redirects for NL and GM levels

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard30**
- Added the new redirect and wildcards for `redirects.js` file.
- Updated the file names.
- Updated all reverse router references and variables.

### CODE REVIEW

File: packages/app/src/next-config/redirects/redirects.js [hunk 1]
```diff
@@ -68,12 +68,40 @@ async function redirects() {
...
+    // Redirects for the archived pages (COR-1420)
+    // NL level
     {
       source: '/landelijk/intensive-care-opnames',
-      destination: `/landelijk/ziekenhuizen-en-zorg`,
+      destination: '/landelijk/ziekenhuizen-en-zorg',
+      permanent: true,
+    },
+    {
+      source: '/landelijk/ziekenhuizen-en-zorg',
```
Comment (VWSCoronaDashboard18) [inline packages/app/src/next-config/redirects/redirects.js:82]: Now on lines 71 and 78, the destination is the source on this line. Update their destination to the same as over here so you won't redirect the user's al around the site...

**Review by VWSCoronaDashboard18 — DISMISSED** [review]
Nice PR. My comment is non-blocking! But might still be worth the attention.

File: packages/app/src/next-config/redirects/redirects.js (hunk 1 above)
Comment (VWSCoronaDashboard30) [reply packages/app/src/next-config/redirects/redirects.js:82]: Fixed!

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Looks good!

**Pull request merged**